innopoints/core/file_manager/base.py

```diff
@@ -0,0 +1,23 @@
+"""Provides a base interface for the file management modules to implement"""
+from abc import ABC, abstractmethod
+from werkzeug.datastructures import FileStorage
+
+
+class FileManagerBase(ABC):
+    """Base abstract class as an interface for file managers"""
+    @abstractmethod
+    def __init__(self, url = ''):
```

Comment:
Spaces around the `=` in parameters, ouch

Reply:
ooh. I guess it's the habit of always surrounding `=` with spaces. I didn't notice 😅